scripts/deploy-v2.js

```diff
@@ -133,7 +133,7 @@ async function main() {
    const beproToken = new ERC20(web3Connection, beproAddress);
    await beproToken.start();
 
-   if(argv.network !== 'custom')
+   if(argv.network !== 'custom' || argv.network !== 'local')
```

Comment:
why do we want to _not_ transfer when we are on a local network ?

Reply:
because these are the test wallets on the test network, on the local network these wallets do not have access to the ganache, so it makes no sense to transfer to them.